e2e/__tests__/clearRequireCache.test.ts

```diff
@@ -0,0 +1,6 @@
+import runJest from '../runJest';
+
+test('clear require cache', () => {
+  const {exitCode} = runJest('clear-require-cache');
```

Comment:
instead of checking in a test, I think this is a good one to just write create a temp directory and use https://github.com/facebook/jest/blob/199f9811ae68b15879cbe18b7ef7ebd61eefcf23/e2e/Utils.ts#L91-L107 to update